inspirehep/modules/migrator/tasks.py

```diff
@@ -125,19 +125,39 @@ def migrate_broken_records(broken_output=None, dry_run=False):
 
 
 @celery.task(ignore_result=True)
-def migrate(source, broken_output=None, dry_run=False):
+def migrate(source, broken_output=None, dry_run=False, wait_for_results=False):
     """Main migration function."""
+    from celery.task.sets import TaskSet
     if source.endswith('.gz'):
         fd = gzip.open(source)
     else:
         fd = open(source)
 
+    if wait_for_results:
+        tasks = []
+        migrate_chunk.ignore_result = False
```

Comment:
Can you add a comment explaining this trick? So that we don't forget in the future.